code/game/response_team.dm

```diff
@@ -22,137 +12,83 @@ var/can_call_ert
 	if(ticker.current_state == 1)
 		to_chat(usr, "<span class='danger'>The round hasn't started yet!</span>")
 		return
-	if(send_emergency_team)
-		to_chat(usr, "<span class='danger'>[GLOB.using_map.boss_name] has already dispatched an emergency response team!</span>")
-		return
 	if(alert("Do you want to dispatch an Emergency Response Team?",,"Yes","No") != "Yes")
 		return
 
-	var/decl/security_state/security_state = decls_repository.get_decl(GLOB.using_map.security_state)
-	if(security_state.current_security_level_is_lower_than(security_state.high_security_level)) // Allow admins to reconsider if the alert level is below High
-		switch(alert("Current security level lower than [security_state.high_security_level.name]. Do you still want to dispatch a response team?",,"Yes","No"))
-			if("No")
-				return
-	if(send_emergency_team)
-		to_chat(usr, "<span class='danger'>Looks like somebody beat you to it!</span>")
+	if(ticker.mode.waiting_for_candidates)
+		to_chat(src, "<span class='warning'>Please wait for the current beacon to be finalized.</span>")
 		return
 
-	var/choice = input(usr, "Select type of strike team:") as null|anything in list("Earth Defence Force", "Unitologists", "Kellion Response Team", "Random")
+	if(ticker.mode.picked_call)
+		ticker.mode.picked_call.reset()
+		ticker.mode.picked_call = null
 
-	switch(choice)
-		if("Earth Defence Force")
-			current_ert = GLOB.ert
-		if("Unitologists")
-			current_ert = GLOB.uni_ert
-		if("Kellion Response Team")
-			current_ert = GLOB.kellion
-		else
-			current_ert = 0
-	message_admins("[key_name_admin(usr)] is dispatching an Emergency Response Team. ([choice])", 1)
-	log_admin("[key_name(usr)] used Dispatch Response Team. ([choice])")
-	trigger_armed_response_team(1)
+	var/list/list_of_calls = list()
+	for(var/datum/emergency_call/L in ticker.mode.all_calls)
+		if(L.name)
+			list_of_calls += L.name
 
-client/verb/JoinResponseTeam()
```

Comment:
Removing this proc? Does that mean joining an ERT is automatic based on antag preferences now?

Reply:
I did not delete it, you can view file.